### PR TITLE
Closes #450 and #451

### LINKED ISSUE #450: There should be a STREFA WOLONTARIUSZA section in the footer
The last section in the footer should be renamed from OFERTY to STREFA WOLONTARIUSZA with sub-sites for volunteers. 
The section should contain:
Załóż konto (http://volontuloapp.org/register)
Lista aktualnych ofert (http://volontuloapp.org/offers)
Archiwum ofert (http://volontuloapp.org/offers/archived)
Lista organizacji i instytucji (http://volontuloapp.org/organizations)
FAQ dla wolontariuszy (http://volontuloapp.org/pages/faq-volunteers)
Przyjdź na dyżur (http://volontuloapp.org/office)

screenshot-at-11-00-50

### PR DESCRIPTION
**Pull request by KrzysztofMadejski**
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/450
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/451
**Reference person:** @krzysztofmadejski
**Description:**
- Added office subpage
- Added footer section:  STREFA WOLONTARIUSZA

**Unit Tests:**
- `apps/volontulo/tests/views/test_pages.py` modified

**What tests do I need to run to validate this change:**
- Go to http://127.0.0.1:8000/
- Navigate to `Przyjdź na dyżur` link in `STREFA WOLONTARIUSZA` footer section
- Should redirect to http://127.0.0.1:8000/office with information about office address and working hours

### CODE REVIEW

**Comment by KrzysztofMadejski** [comment]
Needs a bit of love on the design side.

File: apps/volontulo/tests/views/test_pages.py [hunk 1]
```diff
@@ -100,3 +100,10 @@ def test__get_regulations_staticpage(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'pages/regulations.html')
         self.assertContains(response, u'Regulamin')
+
+    def test_office_subpage(self):
+        u"""Test office subpage."""
+        response = self.client.get('/office')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'pages/office.html')
+        self.assertContains(response, u'Dyżury dla wolontariuszy oraz organizacji')
```
Comment (tomaszwozniak) [inline apps/volontulo/tests/views/test_pages.py]: missing new line at the end file

**Comment by tomaszwozniak** [comment]
Unnecessary u" but I won't block this PR. Rest looks good

**Comment by tomaszwozniak** [comment]
@KrzysztofMadejski please update the branch

**Comment by msobas** [comment]
:+1:

**Comment by KrzysztofMadejski** [comment]
@magul prośba o merge

**Pull request merged**